.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vyos-1x"]
+	path = vyos-1x
```

Comment:
Do we really wan't to have vyos-1x submodule in the toplevel directory? A better place most likely would be under packages.

Reply:
Or would it be an option to clone the repo during build into the background to a well known location? e.g. under build?

Reply:
I'll move the submodule to `packages/` and add a `git submodule update --init` in the background to free the user from having to clone it manually.